src/scene.lisp

```diff
@@ -141,3 +150,32 @@
         (:later `(unload-scene-all-later ,scene))
         (:never nil)
         (t (error ":FREE must be :NOW, :LATER, or :NEVER")))))
+
+(defparameter *scene* nil) ; the current scene in the game loop
+
+;; TODO Perhaps make this generic so users can define :before, :after, :around methods?
```

Comment:
On second thought, this should definitely be generic for the same reason `setup-scene`/`tear-down-scene` are. User overriding and defining methods on their own scene types inheriting from `game-scene`.